src/Form/FormAnalyzer.js

```diff
@@ -246,9 +247,13 @@ class FormAnalyzer {
                         }
                     }
                 )
+            } else {
+                // Here we don't think this is a submit, so if there is another submit in the form, flip the score
+                const thereIsASubmitButton = Boolean(this.form.querySelector('input[type=submit], button[type=submit]'))
+                shouldFlip = thereIsASubmitButton
             }
-            const strength = likelyASubmit ? 20 : 2
-            this.updateSignal({string, strength, signalType: `submit: ${string}`})
+            const strength = likelyASubmit ? 20 : 4
+            this.updateSignal({string, strength, signalType: `button: ${string}`, shouldFlip})
```

Comment:
This just takes into account forms with multiple buttons where only one is obviously a submit. Fixes a long-standing bug for Twitch, already in the test suite 💪.